rambo/saltstack/states/vpn/before.rules

```diff
@@ -0,0 +1,87 @@
+{% set net_dev = salt['cmd.run']('bash /vagrant/salt_resources/states/vpn/dev.sh') %}
```

Comment:
If dev.sh is just one line, why not just put it right here? It'd remove an extra step and a whole file.
```
salt['cmd.run']("echo `ip route | grep default | awk '{print $NF}'`")
```

Reply:
the salt cmd.run command has some real trouble with pipes. some, we needed to move this complex command off inot it's own script